README.md

```diff
@@ -1 +1,63 @@
-# To-Do-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+>This is a website that allows users to manage the tasks. users can add every task to do and when the task is done users can delete the task
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Linters
+- webpack
+
+## Live Demo
+
+[Live Demo Link]()
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+  - IDE eg (Atom/Visual studio code)
+  - Set-up linter(HTML and CSS)
+  - webpack
+
+### Setup
+  - Clone the github repository 
+  - set up HTML, CSS, JavaScript linters 
+  - set up webpack
+
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you add some hints on how setup your projects in the `## Getting Started` section(i.e.: how to clone the project, access the cloned directory,... ) to make it professional.

-  [ **OPTIONAL** ] Also make sure you remove the empty code lines in your code base to make it professional and for best practices.